service/src/integrationTest/kotlin/fi/espoo/evaka/calendarevent/CalendarEventServiceIntegrationTest.kt

```diff
@@ -1648,6 +1648,123 @@ class CalendarEventServiceIntegrationTest : FullApplicationTest(resetDbBeforeEac
         }
     }
 
+    @Test
+    fun `content timestamp updated on all edits except reservation changes`() {
+
+        val form =
+            CalendarEventForm(
+                unitId = testDaycare.id,
+                tree = null,
+                title = "Unit-wide event",
+                description = "uwe",
+                period = FiniteDateRange(today, today.plusDays(3))
+            )
+        val eventId = calendarEventController.createCalendarEvent(dbInstance(), admin, clock, form)
+        val newEventData =
+            calendarEventController.getCalendarEvent(dbInstance(), admin, clock, eventId)
```

Comment:
just as a side note, it's been recommended to write private wrapper functions for the controller calls in the test file, the idea being that the test is easier to read when there's e.g.
`getCalendarEvent(eventId)`
instead of 
`calendarEventController.getCalendarEvent(dbInstance(), admin, clock, eventId)`

Reply:
Cleaned up controller calls.